hanabi_lib/hanabi_hand.h

```diff
@@ -69,7 +72,14 @@ class HanabiHand {
     int Color() const { return color_.Value(); }
     // Returns true if we have no hint saying card is not the given color.
     bool ColorPlausible(int color) const { return color_.IsPlausible(color); }
-    void ApplyIsColorHint(int color) { color_.ApplyIsValueHint(color); }
+    void ApplyIsColorHint(int color) {
```

Comment:
Assume that a player knows 100% it is rainbow if you got two different colour hints. 

I couldn't think of a way to do it in the agent layer.